File: DataStructure/01_Arrays/Concepts/Python/01_BasicOperations.py
arr = [10,20,30,40]

#Insert at the end O(1)
arr.append(50)

#Insert at index 2 O(n)
arr.insert(2, 25)

#Delete last element O(1)
arr.pop()

#Delete element at index 1 O(n)
del arr[1]

#Output : [10, 25, 30, 40]
print(arr)

File: DataStructure/01_Arrays/Concepts/Javascript/01_BasicOperations.js
let arr = [10, 20, 30, 40, 50];

//Insert at the end (O(1))
arr.push(60);

//Insert at index 2 (O(n))
arr.splice(2, 0, 25);

//Delete the last element (O(1))
arr.pop();

//Delete an element at index 1 (O(n))
arr.splice(1, 1);

console.log(arr);
